pkg/cmd/step/verify/step_verify_preinstall.go

```diff
@@ -760,25 +755,6 @@ func (o *StepVerifyPreInstallOptions) gatherGitRequirements(requirements *config
 	return nil
 }
 
-func (o *StepVerifyPreInstallOptions) verifyPrivateRepos(requirements *config.RequirementsConfig) error {
-	log.Logger().Infof("Environment repos will be %s, if you want to create %s environment repos, please set %s to %s in jx-requirements.yml", util.ColorInfo("private"), util.ColorInfo("public"), util.ColorInfo("environmentGitPublic"), util.ColorInfo("true"))
```

Comment:
I feel like we should still be logging this in the else above.

Reply:
ok, will do